blueocean-core-js/src/js/utils/serverBrowserTimeHarmonize.js

```diff
@@ -14,7 +14,21 @@ export class TimeManager {
         return moment();
     }
 
-    harmonizeTimes(props, skewMillis) {
+    format(millis, hintFormat) {
+        return moment.duration(millis).format(hintFormat);
```

Comment:
Should this do the same thing the header and elsewhere does (presumably from the JDL?)